core/init.c

```diff
@@ -36,6 +36,14 @@
 #include <auto_init.h>
 #endif
 
+#ifndef CONFIG_BOOT_MSG
+#define CONFIG_BOOT_MSG 1
+#endif
+
+#ifndef CONFIG_BOOT_MSG_STRING
+#define CONFIG_BOOT_MSG_STRING "main(): This is RIOT! (Version: " RIOT_VERSION ")"
+#endif
```

Comment:
Any reason why those are not exposed in a header?

Reply:
Should they be in a header? They are only used in this file.

Reply:
But only headers are parsed by doxygen, so without moving them to a header they won't be very visible.

Reply:
Ok, moved it to `kernel_init.h` (only the doxygen define, I think it makes sense to keep the message itself local to the file)